### PR TITLE
correct regressions on sub-menu for account and admin

### PR DESCRIPTION
**Pull request by kimsible**
Fixes : 

image
image

### CODE REVIEW

File: client/src/sass/application.scss [hunk 1]
```diff
@@ -145,7 +149,11 @@ label {
     padding-left: $not-expanded-horizontal-margins;
     padding-right: $not-expanded-horizontal-margins;
     position: fixed;
-    z-index: 1;
+    z-index: 200;
```
Comment (rigelk) [inline client/src/sass/application.scss:152]: please use the `z` function, to have z-indexes relative to https://github.com/Chocobozzz/PeerTube/blob/2e7f262724dd64a209e0bad5930ba29bb4f801c3/client/src/sass/include/_variables.scss#L123-L139

**Comment by rigelk** [comment]
This PR breaks account settings.

Screenshot_2020-07-24 Account settings - PeerTube

**Comment by test2a** [comment]
https://ibb.co/d4cLZkj
hey. just found this visual bug on peertub2.cpy.re on my mobile phone just minutes ago.
observations:

    the stats page does not expand to 100%, as you mentioned.
    the header has what appears to be fixed background and that hides the text and other elements as it sticks above all.
    if i resize the browser window, the sidebar does NOT appear in the sidebar on a phone but i think it assumes i have a different resolution and shows up as a smaller window which is not intuitive

**Comment by kimsible** [comment]
> This PR breaks account and admin settings.

Thanks @rigelk =) it's fixed !

**Comment by rigelk** [comment]
Thanks @kimsible :)

**Pull request merged**